index.html

```diff
@@ -38,5 +38,118 @@ <h2 class="text-container" id="sdsection">
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories<br>Gain+Glory</h2>
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <button class="see-project">See Project</button>
+      </div> 
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories<br>Gain+Glory</h2>
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <button class="see-project">See Project</button>
+      </div>
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories<br>Gain+Glory</h2>
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <button class="see-project">See Project</button>
+      </div>
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories<br>Gain+Glory</h2>
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <button class="see-project">See Project</button>
+      </div> 
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories<br>Gain+Glory</h2>
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <button class="see-project">See Project</button>
+      </div> 
+    </div>
+    <section class="about-me">
+      <h2 class="about-me-title">About me</h2>
+      <p class="about-me-description"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+      <button class="resume-button">Get My Resume</button>
+    </section>  
+    <ul class="assets-list">  
+    <div class="card-2-container">
+    <li class="card-2">
+      <img class="icon-lang" src="icon-lang.png" />
+      <h2 class="card-name-2">Languages</h2>
```

Comment:
- [ ] [OPTIONAL] I think you can improve on the design that you have now to make it it more appealing to visitors/potential recruiters or give your users more thrills of a good aesthetic crafting if you can work on separation this two colliding sessions with some spacing elements. Kindly check the shots 👇 

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/213839634-eb495ff8-2166-455e-9fd6-a66326dfaff7.png)|![image](https://user-images.githubusercontent.com/99715640/213839653-7eb02944-a403-4df1-a675-005be6ed364b.png)|